src/shared/common/components/summary/actions/action-summary-interests.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const renderRow = ({ label, value }) => {
+    return (
+        <tr key={label} className='govuk-table__row'>
+            <th className='govuk-table__header padding-left--small govuk-!-width-one-third'>{label}</th>
+            <td className='govuk-table__cell'>{value}</td>
+        </tr>
+    );
+};
+
+renderRow.propTypes = {
+    label: PropTypes.string.isRequired,
+    value: PropTypes.string.isRequired,
+    items: PropTypes.array.isRequired
+};
+
+const ActionSummaryInterests = (props) => {
+
+    const { items: interests } = props;
+
+    if (interests.length < 1) {
+        return false;
+    }
+
+    return (
+        <>
+            <h2 className='govuk-heading-m'>Interest</h2>
+            { interests.map((item, index) => (
+                <>
+                    <table className='govuk-table margin-left--small'>
```

Comment:
I do wonder if pulling Interest out into its own component would be beneficial? @robertdeniszczyc2 what's your opinion?

Reply:
Yeah that could work, small single responsibility components are always preferable :)

Reply:
I've lifted this and the corresponding appeals code into their own components